cardano-cli/src/Cardano/CLI/Shelley/Commands.hs

```diff
@@ -238,14 +239,14 @@ data TransactionCmd
   | TxCalculateMinFee
       TxBodyFile
       (Maybe NetworkId)
-      ProtocolParamsFile
+      ProtocolParamsSource
       TxInCount
       TxOutCount
       TxShelleyWitnessCount
       TxByronWitnessCount
   | TxCalculateMinRequiredUTxO
       AnyCardanoEra
-      ProtocolParamsFile
+      ProtocolParamsSource
```

Comment:
The problem here is `TxCalculateMinRequiredUTxO` is an offline cli command i.e does not necessitate having a running node in the background.